scripts/install

```diff
@@ -166,22 +166,21 @@ ynh_replace_string --match_string="__PHPVERSION__" --replace_string="$phpversion
 #=================================================
 # SECURE FILES AND DIRECTORIES
 #=================================================
-ynh_script_progression --message="Securing files and directories..."
+ynh_script_progression --message="Securing files and directories..." --weight=1
 
 # Set permissions to app files
-chown -R "$app": "$data_path"
-chown -R "$app": "$final_path"
+chown -R $app: "$data_path"
+chown -R $app: "$final_path"
 
 #=================================================
 # SETUP SSOWAT
 #=================================================
-ynh_script_progression --message="Configuring SSOwat..."
+ynh_script_progression --message="Configuring permissions..." --weight=1
 
 # Make app public if necessary
-if [ "$is_public" -eq 1 ]
+if [ $is_public -eq 1 ]
 then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set --app="$app" --key=unprotected_uris --value="/"
+	ynh_permission_update --permission="main" --add="visitors"
```

Comment:
you should remove L53 `ynh_app_setting_set --app="$app" --key=is_public --value="$is_public"`